docs/src/api/class-androiddevice.md

```diff
@@ -358,7 +358,8 @@ This method waits until [AndroidWebView] matching the [`option: selector`] is op
 
 ### param: AndroidDevice.webView.selector
 - `selector` <[Object]>
-  - `pkg` <[string]> Package identifier.
+  - `pkg` <[string]> Optional Package identifier.
+  - `socketName` <[string]> Optional webview socket name.
```

Comment:
It is always good when a single PR does one thing, please split this into a PR that omits driver installation and another PR that allows attaching to a custom socket name.